docs/changes.rst

```diff
@@ -44,13 +44,22 @@ Standard library changes
 .. currentmodule:: amaranth.lib
 
 * Added: :py:`payload_init=` argument in :class:`amaranth.lib.stream.Signature`.
+* Changed: (deprecated in 0.5) providing :meth:`io.PortLike.__add__` is now mandatory. (`RFC 69`_)
```

Comment:
```suggestion
* Changed: (deprecated in 0.5.1) providing :meth:`io.PortLike.__add__` is now mandatory. (`RFC 69`_)
```

Reply:
I've thought about this but then we should probably also edit the other version references to say .0 at the end...

Reply:
Done that.